src/pages/index.tsx

```diff
@@ -19,6 +23,23 @@ function Page(props: Props) {
 
   const title = site?.siteMetadata?.title ?? ''
   const siteUrl = `https://${host}${pathname}`
+  const perPage = 10
+  const productList = useProductsQuery({
+    first: perPage,
+    after: (perPage * 1).toString(),
+    sort: 'score_desc',
+    term: '',
+    selectedFacets: [{ key: 'category-1', value: 'office' }],
+  })
+
+  const products = useMemo(
+    () => productList?.edges.map((edge) => edge.node),
+    [productList]
+  )
+
+  if (products == null) {
+    return null
```

Comment:
Will this make the whole component not render when no products are returned? It seems too drastic if that's the case. Can't we scope this to where `products` is used (in `ProductTiles` below) so it only affects that area of the page? Something like `products && <ProductTiles products={products} />`?

Reply:
You are correct! It looks so drastic hahaha. I'm gonna change it. Thanks!